services_app/src/main/java/org/opendatakit/services/utilities/Constants.java

```diff
@@ -11,4 +11,5 @@
     public static final String ODK_CLIENT_API_VERSION = "2";
     public static final String KEY_LAST_SYNC_INFO = "lastSyncInfo";
     public static final String DATE_TIME_FORMAT = "dd-MM-yyyy HH:mm:ss";
+    public static final String KEY_SYNC_ATTACHMENT_STATE = "syncTypeSpinnerPreference";
```

Comment:
This and `KEY_LAST_SYNC_INFO` should be in https://github.com/odk-x/androidlibrary/blob/development/androidlibrary_lib/src/main/java/org/opendatakit/properties/CommonToolProperties.java.

Please open a pull request to https://github.com/odk-x/androidlibrary first.

Reply:
Thanks for pointing these.
Should I name the value to "common.lastSyncInfo" or "services.lastSyncInfo" and same of spinner. How is the naming being done?